src/chttpd_db.erl

```diff
@@ -733,7 +741,17 @@ db_doc_req(#httpd{method='POST', user_ctx=Ctx}=Req, Db, DocId) ->
     NewDoc = Doc#doc{
         atts = UpdatedAtts ++ OldAtts2
     },
-    case fabric:update_doc(Db, NewDoc, Options) of
+    couch_log:debug("chttpd_db: couchNew doc formed including new att: formed",[]),
+    lists:foreach(fun(X) -> couch_log:debug("~p~n ~p~n",[couch_att:fetch(data, X), couch_att:fetch(name, X)]) end, NewDoc#doc.atts),
```

Comment:
Please, don't do this: if document has a lot of attachments, this loop will spin without producing any useful work, but only slow down all around.